spec/helper/revenuecat_internal_helper_spec.rb

```diff
@@ -361,9 +361,10 @@ def setup_stubs
           body: 'fake-changelog',
           repo: 'RevenueCat/fake-repo-name',
           head: 'fake-current-branch',
-          api_url: 'https://api.github.com'
+          api_url: 'https://api.github.com',
+          labels: 'label_1, label_2'
```

Comment:
I believe this is supposed to be an array of strings 

https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/create_pull_request.rb#L195

```suggestion
          labels: %w[label_1, label_2]
```

Reply:
Oooh I misread this [here](https://github.com/marketplace/actions/create-pull-request). Thanks!